cmd/node/main.go

```diff
@@ -1250,9 +1234,7 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 		return fmt.Errorf("%w when adding nodeShufflerOut in hardForkTrigger", err)
 	}
 
-	var elasticIndexer indexer.Indexer
-	if !check.IfNil(coreServiceContainer) && !check.IfNil(coreServiceContainer.Indexer()) {
-		elasticIndexer = coreServiceContainer.Indexer()
+	if !elasticIndexer.IsNilIndexer() {
 		elasticIndexer.SetTxLogsProcessor(processComponents.TxLogsProcessor)
```

Comment:
we need to rethink the Indexer interface - have one common - which gets all the data -> after that we could RegisterHandlers. One of those can be elastic search.

Reply:
I will try to do some small, small redesign when implementing the `ScWatcher`. Will see how it works.